dydx/dydxFormatter/dydxFormatter/_Utils/dydxFeatureFlag.swift

```diff
@@ -15,6 +15,7 @@ public enum dydxBoolFeatureFlag: String, CaseIterable {
     case force_mainnet
     case enable_trading_rewards
     case enable_reduce_only
+    case disable_app_rating
```

Comment:
added a feature flag, will want to monitor pre-prompt events to make sure there are no bugs. Even though tests should be comprehensive, better to be safe.